src/bedita-api-client.ts

```diff
@@ -216,7 +216,17 @@ export class BEditaApiClient {
             }
         }
 
-        return this.#axiosInstance.interceptors.response.eject(id);
+        this.#axiosInstance.interceptors.response.eject(id);
+    }
+
+    /**
+     * Return the request interceptors map
+     *
+     * @returns Map<string, number>
```

Comment:
At the moment we haven't write `@return` annotation in doc block since it could be considered unnecessary because of Typescript return types.
I'm not  entirely convinced but for now we could remove it to maintain the same convention.

Reply:
https://github.com/atlasconsulting/bedita-sdk-js/commit/74a9a2174982fbea109403ebf378c9a7c4e37d24